osf/models/files.py

```diff
@@ -615,7 +621,7 @@ class FileVersion(ObjectIDMixin, BaseModel):
     identifier = models.CharField(max_length=100, blank=False, null=False)  # max length on staging was 51
 
     # Date version record was created. This is the date displayed to the user.
-    date_created = NonNaiveDateTimeField(auto_now_add=True)
+    date_created = NonNaiveDateTimeField(auto_now_add=True, db_index=True)
```

Comment:
Why was `db_index=True` added? Do we need an ascending index? I assume not, since there isn't a migration for (though running `makemigrations` results in a new migration).

Reply:
Oops, good call, forgot to remove that when we decided to add the negative index instead!